recipes/speex/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2rc2" %}
```

Comment:
@jakirkham, @mcernak is trying to add `kaldi` to conda-forge. Since kaldi requires this version: https://github.com/kaldi-asr/kaldi/blob/85a3dd5f0b71e419abf1169a26b759bfc423a543/tools/extras/install_speex.sh, I think this is why he added this version. Since Kaldi has 0 testing or any release, users are forced to do follow exactly the same guidelines.